components/scene change/StartNewDeckSetupWizard.h

```diff
@@ -1,14 +1,24 @@
 #pragma once
 #include "kitten/mouse picking/ClickableUI.h"
+#include "kitten\InputManager.h"
+
 #include "_Project\StringInputDisplay.h"
 
-class StartNewDeckSetupWizard : public kitten::K_Component
+#include <string>
+
+class StartNewDeckSetupWizard : public kitten::K_Component, public input::StringListener
```

Comment:
Dang now I realize that I wasn't properly using StringListener, I was adding it as a separate component when I could have been overriding the functions instead....